NSS-Day-Cohort-66/rare-api-golden-droids
# destroy comment added

added `destroy` method to Comment View

**To Test**

1. switch to main and fetch
2. change to `zd-destroy-comment`
3. open Postman 
4. change method to DELETE with the request `http://localhost:8000/comments/n`
5. use the following token
```
Token 978afa7b76527cc21d76d7b5430ab77f73aa3bff
```
6. you should receive a `204 no content`
7. now change token to 
```
Token fa2eba9be8282d595c997ee5cd49f2ed31f65bed
``` 
8. and send DELETE request to `http://localhost:8000/comments/4`
9. you should receive a `204 no content`
10. change request to `http://localhost:8000/comments/2`
11. you should receive a 403 forbidden and a response of 
```
"message": "You don't have admin rights!"
```
